compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/cfa/FirCallsEffectAnalyzer.kt

```diff
@@ -65,7 +65,16 @@ object FirCallsEffectAnalyzer : FirControlFlowChecker(MppCheckerKind.Common) {
 
         for ((symbol, firEffect) in argumentsCalledInPlace) {
             val requiredRange = (firEffect.effect as ConeCallsEffectDeclaration).kind
-            val foundRange = invocationData.getValue(graph.exitNode)[NormalPath]?.get(symbol)?.withoutMarker ?: EventOccurrencesRange.ZERO
+
+            // We map `EventOccurrencesRange` to `InvocationKind`, as the latter has fewer
+            // kinds, so we would suggest using a non-existent `InvocationKind` to the user.
```

Comment:
Extending `InvocationKind` would probably be a better solution and would allow a more fine-tuned communication between the users and the compiler, though I'm not sure that change wouldn't have to go through the language design team first.

Reply:
Yeah, there are no plans to expand values of `InvocationKind`, so your approach is fine